contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -633,58 +643,267 @@ contract LatestConfigDigestAndEpoch is TransmittedDualAggregatorBaseTest {
     assertEq(epoch, 1, "epoch not correct");
   }
 }
-contract LatestAnswer is TransmittedDualAggregatorBaseTest {
-  function test_ReturnsLatestAnswer() public view {
-    assertEq(aggregator.latestAnswer(), 1);
+
+// latestAnswer(): test primary and secondary caller
+contract LatestAnswer is ConfiguredDualAggregatorBaseTest {
```

Comment:
i'd love to see all of this shared `setUp` code pulled out into a new contract that can be extended by all of the tests that need it.

for example:
```solidity
// pseudo code
contract RoundDataDualAggregatorBaseTest is ConfiguredDualAggregatorBaseTest {
  uint256 cutoffTime = 21;
  uint256 latestRoundIds = 6;
  uint256 startingBlock = 31;
  address secondaryCaller = address(102);

  function setUp() public override {
    super.setUp();
    harness.injectTransmissions(); // see other comment about improving this
    harness.setLatestRoundIds(latestRoundIds);
    harness.setCutoffTime(cutoffTime);
    vm.warp(startingBlock);
  }
}```

then you can just tweak the instance variables for each test.